src/modules/operate/AccountOperation.ts

```diff
@@ -381,6 +392,117 @@ export class AccountOperation {
     return this;
   }
 
+  public addSignedOperation(
+    signedOperation: SignedOperation,
+  ): AccountOperation {
+    // store the proof
+    this.proofs.push({
+      startIndex: new BigNumber(this.actions.length),
+      numActions: new BigNumber(signedOperation.actions.length),
+      salt: signedOperation.salt,
+      expiration: signedOperation.expiration,
+      sender: signedOperation.sender,
+      typedSignature: signedOperation.typedSignature,
+    });
+
+    // store the actions
+    for (let i = 0; i < signedOperation.actions.length; i += 1) {
+      const action = signedOperation.actions[i];
+      const nullData =
+        (action.data === '0x') ||
+        (action.data === null) ||
+        (action.data === undefined);
+
+      const secondaryAccountId = action.secondaryAccountOwner === ADDRESSES.ZERO
+        ? 0
+        : this.getAccountId(
+          action.secondaryAccountOwner,
+          action.secondaryAccountNumber,
+        );
+
+      this.addActionArgs(
+        {
+          primaryAccountOwner: action.primaryAccountOwner,
+          primaryAccountId: action.primaryAccountNumber,
+        },
+        {
+          actionType: action.actionType,
+          primaryMarketId: action.primaryMarketId.toFixed(0),
+          secondaryMarketId: action.secondaryMarketId.toFixed(0),
+          otherAddress: action.otherAddress,
+          otherAccountId: secondaryAccountId,
+          data: nullData ? [] : toBytes(action.data),
+          amount: {
+            reference: action.amount.ref,
+            denomination: action.amount.denomination,
+            value: action.amount.value.times(action.amount.sign ? 1 : -1),
+          },
+        },
+      );
+    }
+
+    return this;
+  }
+
+  public createOperation(
```

Comment:
I don't understand what this function is for, can you explain?

Reply:
it was for your comment below